skimage/data/__init__.py

```diff
@@ -337,12 +337,16 @@ def _load(f, as_gray=False):
 def camera():
     """Gray-level "camera" image.
 
-    Often used for segmentation and denoising examples.
+    Can be used for segmentation and denoising examples.
 
     Returns
     -------
     camera : (512, 512) uint8 ndarray
         Camera image.
+
+    Notes
+    -----
+    No copyright restrictions. CC0 by the photographer (Lav Varshney).
```

Comment:
Can you add a `.. versionchanged:: 0.18` here explaining the change and linking to the issue? "This image was changed due to copyright restrictions. See [1]_ for more information.", and a References section for [1].

Here's a versionchanged example for reference:

https://github.com/scikit-image/scikit-image/blob/83e857ca8f974bb560315a00989673b4b52f8a93/skimage/exposure/exposure.py#L346-L348

Reply:
Lovely update, thank you! 😊